bin/matrix-search.js

```diff
@@ -24,24 +24,25 @@ Matrix.localization.init(Matrix.localesFolder, Matrix.config.locale, function ()
 
     Matrix.firebase.app.search(needle, function (data) {
       Matrix.loader.stop();
-      if (!_.isNull(data)) {
-        debug(data);
 
-        if ( !_.isArray(data)){
-          data = [ data ];
+      debug(data);
+      if ( _.isNull(data)  || data.meta.visible === false ) {
```

Comment:
Hey Sean Close this PR .. 
I create other PR in the repo `matrix-firebase` fixing this error !